### PR TITLE
Enemies and Allies Content Port

### PR DESCRIPTION
**Pull request by fenndragon**
# Description

This is a batch of changes I made for Floof station being ported over to the upstream branch

---

# Changelog

:cl:
- add: Added new neutral xenos, events, and reagent slimes

### CODE REVIEW

File: Resources/Prototypes/Floof/Body/Organs/Friendstomach.yml
Comment (SleepyScarecrow) [inline Resources/Prototypes/Floof/Body/Organs/Friendstomach.yml:1]: have to remove floof folder

File: Resources/Prototypes/Floof/Body/Prototypes/Friendshaped.yml
Comment (SleepyScarecrow) [inline Resources/Prototypes/Floof/Body/Prototypes/Friendshaped.yml:1]: have to remove floof folder

**Review by SleepyScarecrow — CHANGES_REQUESTED** [review]
just nitpicky directory stuff from me :3

**Review by SleepyScarecrow — APPROVED** [review]

**Review by Mnemotechnician — CHANGES_REQUESTED** [review]
The entire changelog should be just 1-2 entries, and should not include technical details.

**Comment by fenndragon** [comment]
> The entire changelog should be just 1-2 entries, and should not include technical details.

better?

**Comment by Mnemotechnician** [comment]
It should also be descriptive for someone who's reading it in the menu. "Added new neutral xeno prototypes for admins to use in events" or something similar would be good.

**Comment by fenndragon** [comment]
> It should also be descriptive for someone who's reading it in the menu. "Added new neutral xeno prototypes for admins to use in events" or something similar would be good.

I listed a bit better what all was added but I did it in one entry, better?

**Review by Mnemotechnician — APPROVED** [review]
Yea, that's fine now.

**Review by FoxxoTrystan — COMMENTED** [review]
Except for Rouney Sprite, i dont think those additions are needed.

Those Neutral things were things you done mostly for floof, il let solidus view it.

**Comment by fenndragon** [comment]
> Except for Rouney Sprite, i dont think those additions are needed.
> 
> Those Neutral things were things you done mostly for floof, il let solidus view it.

it was requested.

**Comment by fenndragon** [comment]
> Except for Rouney Sprite, i dont think those additions are needed.
> 
> Those Neutral things were things you done mostly for floof, il let solidus view it.

Also, how is additional events and reagent slimes not needed? this is literally just additional content.

**Review by VMSolidus — APPROVED** [review]
Wuh>? This is fine though?

**Comment by VMSolidus** [comment]
Or well, it would be when the Linter fail is fixed. That's legitimate.

**Comment by fenndragon** [comment]
> Or well, it would be when the Linter fail is fixed. That's legitimate.

whoops forgot to change the texture paths, my bad

**Comment by fenndragon** [comment]
went ahead and just added 3 more events and fixed the event text in the locale while I was thinking about it

File: Resources/Prototypes/Entities/Mobs/NPCs/Rslimes.yml [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    - id: ReagentSlimeUnstableMutagen
+      prob: 0.01
+    - id: ReagentSlimeLead
+      prob: 0.01
+    - id: ReagentSlimechlorinetriflouride
+      prob: 0.01
+    - id: ReagentSlimePotassium
+      prob: 0.01
+    - id: ReagentSlimeLotophagoiOil
+      prob: 0.01
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/Rslimes.yml:308]: Im not gona request Changes but not gona approve this, i was against this on floof, will be here.

Those are only for anomalies.

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+    - id: MobArgocyteHarvester
+      prob: 0.005
+    - id: MobArgocyteCrawler
+      prob: 0.005
+    - id: MobArgocyteEnforcer
+      prob: 0.005
+    - id: MobArgocyteFounder
+      prob: 0.005
+    - id: MobArgocyteLeviathing
+      prob: 0.005
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:405]: Same thing, those if i recall are used for asteroids/salvage

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+    - id: MobFleshJared
+      prob: 0.005
+    - id: MobFleshGolem
+      prob: 0.005
+    - id: MobFleshClamp
+      prob: 0.005
+    - id: MobFleshLover
+      prob: 0.005
+    - id: MobAbomination
+      prob: 0.005
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:432]: Anomaly Stuff again

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+    startAnnouncement: true
+    earliestStart: 20
+    reoccurrenceDelay: 12
+    minimumPlayers: 1
+    weight: 6 # Really weak compared to other critters
+    duration: 60
+  - type: VentCrittersRule
+    entries:
+    - id: MobTick
+      prob: 0.01
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:449]: Space Stuff

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+    - id: MobCarpMagic
+      prob: 0.01
+    - id: MobCarpHolo
+      prob: 0.01
+    - id: MobCarpRainbow
+      prob: 0.01
+    - id: MobShark
+      prob: 0.01
+    - id: MobCarpDragon
+      prob: 0.01
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:476]: Carps should not come out of vents, pehaps spawn outside of the station but not inside.

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 6]
```diff
@@ -0,0 +1,540 @@
...
+  - type: VentCrittersRule
+    entries:
+    - id: MobBearSpace
+      prob: 0.01
+    - id: MobKangarooSpace
+      prob: 0.01
+    - id: MobSpiderSpace
+      prob: 0.01
+    - id: MobCobraSpace
+      prob: 0.01
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:499]: Bear, Kangaroo, this is again, Salvage spawns.

File: Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml [hunk 7]
```diff
@@ -0,0 +1,540 @@
...
+  - type: VentCrittersRule
+    entries:
+    - id: MobLivingLight
+      prob: 0.01
+    - id: MobLuminousPerson
+      prob: 0.01
+    - id: MobLuminousObject
+      prob: 0.01
+    - id: MobLuminousEntity
+      prob: 0.01
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml:522]: This is again anomly spawns

**Review by FoxxoTrystan — COMMENTED** [review]
New Events in general just remove all the uniquness of anomalies that can spawn it ect...
And some of them does not fit, i would not approve something like this, review goes to others for it.

**Comment by fenndragon** [comment]
> New Events in general just remove all the uniquness of anomalies that can spawn it ect... And some of them does not fit, i would not approve something like this, review goes to others for it.

that's fine, and I was going to leave it at that but it was then requested as a feature personally by others to port it upstream so that's what I'm doing.

**Review by SleepyScarecrow — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Review by SleepyScarecrow — APPROVED** [review]

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**